src/components/ImportAddressesButton.jsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
This used to be the `BitcoindAddressImporter` component. In addition to re-naming, I simplified some of the error/info messages, put the note about rescan's into a tooltip and info button, and converted this to a functional component that uses React Hooks, which I felt simplified the implementation of this component and made state management a little simpler.

This also used to be hooked up to the redux store using `connect` but I felt it was an unnecessary being better to pass in the necessary store items as properties. This made even more sense given that in the two places where this is used (Script Explorer/Interactions and the Wallet) it doesn't need the same things from the state.